policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/living_tax_credit/threshold/middle/base.yaml

```diff
@@ -0,0 +1,9 @@
+description: Northwest Territories cost of living tax credit threshold for middle income person in 2022
```

Comment:
make the whole table from the [pdf](https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5012-tc/5012-tc-22e.pdf) a marginal rate parameter